dist/fluxo.js

```diff
@@ -444,38 +448,5 @@ Fluxo.CollectionStore = Fluxo.Base.extend(
 });
 
 
-  Fluxo.actionHandlers = {};
-
-Fluxo.registerActionHandler = function(identifier, handler) {
-  handler = Fluxo.extend({}, handler);
-
-  Fluxo.actionHandlers[identifier] = handler;
-
-  var args = Array.prototype.slice.call(arguments, 2);
-
-  if (typeof handler.initialize == "function") {
-    handler.initialize.apply(handler, args);
-  }
-};
-
-Fluxo.callAction = function(actionHandlerIdentifier, actionName) {
-  var handler = Fluxo.actionHandlers[actionHandlerIdentifier];
-
-  if (!handler) {
-    throw new Error("Action handler " + actionHandlerIdentifier + " not found.");
-  }
-
-  var action = handler[actionName];
-
-  if (!action) {
-    throw new Error("Action " + actionName + " not found on " + actionHandlerIdentifier + " handler.");
-  }
-
-  var args = Array.prototype.slice.call(arguments, 2);
-
-  return action.apply(handler, args);
-};
-
-
```

Comment:
This was removed because I hasn't bumped a version yet, I build a new dist only on version bump.

I will change this, committing a new dist on every new commit, but for now, to not mess up your commit with unrelated changes can you remove the change on dist file?